package.json

```diff
@@ -1681,5 +1681,6 @@
     "xmlbuilder": "13.0.2",
     "yargs": "^15.4.1",
     "yarn-deduplicate": "^6.0.2"
-  }
+  },
+  "packageManager": "^yarn@1.22.19"
```

Comment:
Apologies, it looks like I was mistaken and the `^` is not used.  We're running 1.22.21 on CI, do you mind if we update the versions to 1.22.21 and remove the ^?

Reply:
Not at all! I can do it too